main.tf

```diff
@@ -2,23 +2,49 @@
 # Launch configuration
 #######################
 resource "aws_launch_configuration" "this" {
-  count = "${var.create_lc}"
+  count = var.create_lc
```

Comment:
should be conditional
 count = var.create_lc ? 1 : 0

Reply:
Fixed